client/dashboard/sites/backups/backups-list.tsx

```diff
@@ -10,6 +10,11 @@ import { useBackupState } from './use-backup-state';
 import type { ActivityLogEntry, Site } from '@automattic/api-core';
 import type { View } from '@wordpress/dataviews';
 
+const DEFAULT_LAYOUTS = {
+	table: {},
+	list: {},
+};
+
```

Comment:
We are not supporting the table view on the Backup page, only `list`.

```suggestion
```

Reply:
Ah okay, `table` was already there so I let it be, but I'll remove it!